Dockerfile

```
@@ -21,7 +21,7 @@ COPY --from=builder1 /app /app
 
 RUN composer create-project prestashop/traces
 
-RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml"
+RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml" --timeout 30000
```

```suggestion
RUN ./traces/traces -u $ENV_USER_LOGIN -p $ENV_USER_PASS -o PrestaShop --config="./traces/config.dist.yml" --timeout 30
```

The timeout is in seconds, not milliseconds.

---

Oh ! That's odd, error message was in ms:
```cURL error 28: Operation timed out after 2000 milliseconds```